crash-handler-process/platforms/process-win.cpp

```diff
@@ -230,6 +230,10 @@ void Process_WIN::terminate(void) {
     log_debug << "Terminated pid : " << PID << std::endl;
 }
 
+void Process_WIN::terminateNicely(void) {
+	this->terminate();
```

Comment:
on Windows, this is the same

Reply:
On Windows, the process check is done by doing a call to `OpenProcess` and keeping a handle on this, and then a thread starts monitoring the handle. When waiting for handle is done, `isAlive` is set to FALSE.

On OSX, periodically, a call to `proc_pidinfo` is made which queries for the process name to start with some agreed upon string. If check fails, process is not alive